src/main/resources/com/google/api/codegen/ruby/sample.snip

```diff
@@ -142,14 +147,45 @@
   @end
 @end
 
+@private methodCall(class)
```

Comment:
I guess I don't really like this because we'd have to duplicate conditional structures to do the page token assignment. Can we leave it as is? (It's actually less repetitive than the corresponding existing code in some other language snippets.)